docs/DeveloperGuide.md

```diff
@@ -1,38 +1,135 @@
-# Developer Guide
+# Developer Guide for SpendSwift
 
 ## Acknowledgements
-
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
-
-## Design & implementation
-
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
-
-
-## Product scope
-### Target user profile
-
-{Describe the target user profile}
-
-### Value proposition
-
-{Describe the value proposition: what problem does it solve?}
+{List here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well.}
+
+## Design & Implementation
+### Sequence Diagrams
+#### add-expense
+![Add Expense Sequence Diagram](diagrams/AddExpense.png)
+
+#### add-category
+![Add Category Sequence Diagram](diagrams/AddCategory.png)
+
+#### delete-expense
+![Delete Expense Sequence Diagram](diagrams/DeleteExpense.png)
```

Comment:
Similarly for the other diagrams as well, for the arrows pointing left (for printing something) I believe they should be dotted as well (refer to the Week 10 notes)